daemon/src/model/cfd.rs

```diff
@@ -593,6 +593,28 @@ impl Cfd {
 
         Ok(signed_refund_tx)
     }
+
+    pub fn filter_pending_open_dlc(&self) -> Option<Dlc> {
+        if let CfdState::PendingOpen { dlc, .. } = self.state.clone() {
+            Some(dlc)
+        } else {
+            None
+        }
+    }
+
+    pub fn filter_must_refund(&self) -> bool {
+        matches!(self.state.clone(), CfdState::MustRefund { .. })
+    }
+
+    pub fn filter_cleanup(&self) -> bool {
```

Comment:
I don't think these would necessarily have to be named `filter_`. They could be used in other contexts as well.

For the latter two, a naming with `is_` would be more idiomatic.

The first one is simply a getter for the the dlc in the `pending_open` state (which is why I proposed `pending_open_dlc` originally). Getters are named without `get_` in Rust.